tests/Avalonia.Skia.UnitTests/Media/FontManagerTests.cs

```diff
@@ -13,6 +14,20 @@ public class FontManagerTests
     {
         private static string s_fontUri = "resm:Avalonia.Skia.UnitTests.Assets?assembly=Avalonia.Skia.UnitTests#Noto Mono";
 
+        [Win32Fact("Only run on Windows")]
```

Comment:
Win32Fact is defined in a different assembly.
For now, you can copy Win32Fact.cs to this assembly...